### PR TITLE
Side events

### PR DESCRIPTION
**Pull request by mermop**
Have edited @niceking's copy down a bit.

- [x] Copy OK.
- [x] Design OK.
- [x] Copy from flux.
- [x] Figure out how to link in menu

### Screenshots

<details>
<summary>Desktop</summary>

screencapture-localhost-4567-events-1504574536447

</details>

<details>
<summary>Mobile</summary>

screencapture-localhost-4567-events-1504574554427
</details>

### CODE REVIEW

File: data/events.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - "We will have an after party for you to talk about the exciting things you learned and thought about at Kiwi Ruby."
+      - "Karaoke may ensue."
+  - name: "🐦🦆 Zealandia tour"
+    time: "Saturday 4 November, 12:45pm"
+    location: "Meet outside St James Theatre, Courtenay Place"
+    description_paragraphs:
+      - "New Zealand is home to a wonderful array of native birds, from the iconic flightless Kiwi that adorns much of our national imagery, to the cheeky kākāpō Sirocco, which you may know in your Slack channels as <a href='http://cultofthepartyparrot.com/'>:partyparrot:</a>."
+      - "If you’re a nature loving Rubyist, you may like to come on a trip to <a href='https://www.visitzealandia.com/'>Zealandia</a>, the world’s first fully-fenced urban ecosanctuary."
+      - "Bring warm clothes and your walking shoes. Zealandia is suitable for the whole family, and has a number of stroller and wheelchair accessible trails."
+    payment: "Bring $30 for admission and bus fare"
```
Comment (mermop) [inline data/events.yml]: @niceking how much actually is this?

File: source/events.html.slim [hunk 2]
```diff
@@ -5,18 +5,19 @@ title: Events
...
-        h3.c-event__details
-          span= event.payment
-          - if event.rsvp_link
-            span.c-event__spacer &nbsp;•&nbsp;
-            a(href=event.rsvp_link)=event.rsvp_text
-        - if event.description_paragraphs
-          - event.description_paragraphs.each do |paragraph|
-            p=paragraph
+        .grid
+          .col-1-2
```
Comment (jasminwong) [inline source/events.html.slim]: To show a 1:3 column structure like on /talks, I think you could use `.col-1-4` here.

File: source/events.html.slim [hunk 3]
```diff
@@ -5,18 +5,19 @@ title: Events
...
+            h3.c-event__details= "📅 #{event.time}"
+            h3.c-event__details= "📍 #{event.location}"
+            h3.c-event__details= "💸 #{event.payment}"
+            - if event.rsvp_link
+              h3.c-event__details
+                a(href=event.rsvp_link)="📬 #{event.rsvp_text}"
+          .col-1-2
```
Comment (jasminwong) [inline source/events.html.slim]: ...and `.col-3-4` here. They will collapse into a single column on mobile.

**Review by jasminwong — COMMENTED** [review]
Suggested some changes on the column structure.

File: source/events.html.slim (hunk 3 above)
Comment (mermop) [reply source/events.html.slim]: Works well for the events with lots of description..
screen shot 2017-09-05 at 1 17 14 pm

less well for the ones with less in the description
screen shot 2017-09-05 at 1 17 06 pm

Could just address that by moving some of the details from the `cost` section into the description though

Comment (mermop) [reply source/events.html.slim]: Yeah, sorted, will do that 👍 
screen shot 2017-09-05 at 1 20 55 pm

File: data/events.yml (hunk 1 above)
Comment (niceking) [reply data/events.yml]: It's $19.50 for entry and the bus is $3.50 each way. I was going to gauge interest for tours on the sign up sheet cos it's cheaper in a group so I'd like the wording to be ambiguous. 

Can you change this copy to read 

Bring money for <a href='https://www.visitzealandia.com/Visit#VisitingHours'>admission</a> and $7 cash (or Snapper) for each return adult bus fare.

Comment (mermop) [reply data/events.yml]: It's a pretty small space so I'm putting that in the main copy and just "Bring admission and bus fare" on the side

screen shot 2017-09-06 at 6 35 23 pm

**Comment by mermop** [comment]
I've just put it in the menu, it breaks onto two lines but `˜\_(°~°)_/˜`

**Pull request merged**